Foundation/CGFloat.swift

```diff
@@ -143,6 +143,10 @@ public struct CGFloat {
     
     /// The native value.
     public var native: NativeType
+
+    public var double: Double {
```

Comment:
@_transparent functions aren't supposed to invoke internal methods. See: https://github.com/apple/swift/blob/master/docs/TransparentAttr.rst

It is *doable* to do this manually, yes? After all, it's just writing `Double(foo.native)` instead of `foo.double`.